src/sig_stfl/sig_stfl.c

```diff
@@ -275,7 +275,11 @@ OQS_API int OQS_SIG_STFL_alg_is_enabled(const char *method_name) {
 	}
 #ifdef OQS_ENABLE_SIG_STFL_LMS
```

Comment:
After you add individual options, you can remove `#ifdef OQS_ENABLE_SIG_STFL_LMS` like XMSS.

Reply:
OK.